pir_frontend/views.py

```
@@ -0,0 +1,36 @@
+import logging
+
+from requests.exceptions import HTTPError
+from django.shortcuts import render
+from pir_frontend.forms import PIRForm
+
+logger = logging.getLogger(__name__)
+
+# Create your views here.
+def form_base(request):
+    form = PIRForm()
+    context = {}
+
+    if request.method == 'POST':
+        form = PIRForm(request.POST)
+        if form.is_valid():
+            data = form.cleaned_data
+
+            try:
+                form.client.create_report(data)
+            except Exception as e:
```

Catching all is not really good practice. The client should return a specific exception. We need to log these exceptions on Sentry